### PR TITLE
Project 1: JS Objects

### PR DESCRIPTION
**Pull request by ookino**
## Features Added 💯 
- Added function to add a new book to an array of objects and local storage
- Added function to remove book from an array of objects and local storage

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello Team 👋,

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
```
Comment (mikenath223) [inline index.html:1]: - [ ] Kindly update the readme using this [template](https://github.com/microverseinc/readme-template).

Comment (ookino) [reply index.html:1]: @mikenath223 changes made

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hi there, 

Awesome job on making the changes requested in the previous review 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. Let's work on those.


 ## Required Changes ♻️
_Kindly check the comments under the review._

Happy coding, Cheers!🚀🚀

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it's valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1,2 +1,48 @@
-# awesome-books
- 
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> My Personal Portfolio
+
+**Additional description about the project and its features.**
```
Comment (mikenath223) [inline README.md]: - [ ] On the readme, please update the project name and description to match this project.

Comment (ookino) [reply README.md]: @mikenath223, i made a mistake and i have fixed the issue, please take a look at it again 
Thank you

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <style>
+      ul {
+        padding: 0;
+        list-style-type: none;
+      }
```
Comment (rustam-tolipov) [inline index.html:12]: Why not you use external CSS, it would make your html more cleaner

**Review by rustam-tolipov — COMMENTED** [review]
I checked your code it looks fine. 
But one optional thing to change.

**Pull request merged**